rule_packages/c/Contracts5.json

```diff
@@ -0,0 +1,38 @@
+{
+  "CERT-C": {
+    "ERR32-C": {
```

Comment:
Im having a moment here, but doesnt including this in the package json usually cause CI failure without the rest of the files? or did we change that a long time ago?